tests/PageObjects/SongListPage.ts

```diff
@@ -108,12 +112,36 @@ export class SongListPagePO {
     await this.page.keyboard.press('Enter');
   }
 
-  public async expectSongToBeVisibleAsNew(songID: string) {
-    await expect(this.page.locator('[data-group-letter="New"]').getByTestId(`song-${songID}-new-group`)).toBeVisible();
-  }
-
   public async goBackToMainMenu() {
     await this.page.waitForTimeout(500);
     await this.page.keyboard.press('Backspace');
   }
+
+  public get selectionPlaylistTip() {
+    return this.page.locator("[role='tooltip']");
+  }
+
+  public async closeTheSelectionPlaylistTip() {
+    await this.selectionPlaylistTip.getByText('Close').click();
```

Comment:
can you add the `data-test` attribute to the button>

Reply:
ok, done :)